example/FFTConvolutionOverRegionOfInterest.ipynb

```diff
@@ -141,17 +141,18 @@
     "\n",
     "\n",
     "def make_image() -> image_type:\n",
-    "    arr = np.ones([100, 100]).astype(np.float32)\n",
+    "    arr = np.ones([150, 150]).astype(np.float32)\n",
     "    arr[20:80, 20:80] = 5\n",
-    "    arr[40:60, 40:60] = 0\n",
+    "    arr[70:120, 60:100] = 0\n",
+    "    arr[30:60,50:120] = 10\n",
     "    image = itk.image_from_array(arr)\n",
     "    return image\n",
     "\n",
     "\n",
     "def make_roi() -> region_type:\n",
     "    region = itk.ImageRegion[2]()\n",
-    "    region.SetSize([40, 40])\n",
-    "    region.SetIndex([30, 30])\n",
+    "    region.SetSize([80, 50])\n",
+    "    region.SetIndex([30, 50])\n",
```

Comment:
If index `[30, 60]` is used, does any yellow creep into the RoI? I expect that it would with requested region approach, but not with sub-region.

Reply:
Yes, testing with the index `[30,60]` shows that the yellow region (assigned by matplotlib to the pixel intensity "10") blurs into the requested output region for demonstrated spatial and FFT approaches, with once again negligible output difference.

I have not demonstrated in this notebook, but I agree that if we cropped the input image to the output region before blurring we would not see the yellow region blur into the final product because that signal would be completely removed from input.

Reply:
It feels like you are trying to prove that the two approaches are equivalent. I much prefer to show that they are not equivalent, so care is needed if substituting one for the other.

Reply:
The point I am trying to make in this notebook is how we can draw equivalence between spatial and FFT convolution methods, specifically in the case of different subregions padded for the kernel width because that is the main place where `itk.FFTConvolutionImageFilter` and `itk.ConvolutionImageFilter` differ in their existing implementation.

If it would be helpful for discussion I can add an additional block comparing convolution results when applied to an image with versus without additional padding for the kernel image. However the difference of those results is not limited to either spatial or FFT approaches to convolution.

Reply:
Oh, you are right. I forgot that the difference comes from padding, not the algorithm per se.